docs/ABI/Mangling.rst

```diff
@@ -216,19 +216,28 @@ Entities
   ADDRESSOR-KIND ::= 'p'                     // pinning addressor (native owner)
 
   decl-name ::= identifier
-  decl-name ::= identifier 'L' INDEX         // locally-discriminated declaration
-  decl-name ::= identifier identifier 'LL'   // file-discriminated declaration
+  decl-name ::= identifier 'L' INDEX                  // locally-discriminated declaration
+  decl-name ::= identifier identifier 'LL'            // file-discriminated declaration
+  decl-name ::= identifier 'L' RELATED-DISCRIMINATOR  // related declaration
+
+  RELATED-DISCRIMINATOR ::= [a-j]
+  RELATED-DISCRIMINATOR ::= [A-J]
```

Comment:
Please document the fact that you've claimed some of these here.

Reply:
I don't consider them "claimed". Their meaning is context-dependent. "The Clang importer is using 'LE' and 'Le' for top-level values under the 'SC' module" doesn't seem like part of the mangling schema or general ABI to me.